Xamarin.Forms.Core/IDispatcherProvider.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Xamarin.Forms
+{
+	public interface IDispatcherProvider
```

Comment:
why do we need this to be public / if it's for the platform code only, hide it from intelisense

Reply:
Fair enough, I've added the EditorBrowseable.Never.